packages/react/src/component-tests/IcPopoverMenu/IcPopoverMenuData.tsx

```diff
@@ -0,0 +1,291 @@
+        </IcButton>
+        <IcPopoverMenu
+          anchor="popover-button"
+          aria-label="popover"
+          onIcPopoverClosed={() => handleClose()}
+          ref={popoverEl}
+        >
+          <IcMenuItem disabled={true} label="Copy code"></IcMenuItem>
+          <IcMenuItem disabled={true} label="Paste code"></IcMenuItem>
+          <IcMenuItem
+            disabled={true}
+            label="Actions"
+            submenuTriggerFor="actions"
+          />
+        </IcPopoverMenu>
+        <IcPopoverMenu submenuId="actions" aria-label="popover">
+          <IcMenuItem label="Edit" />
+          <IcMenuItem label="Find" />
+          <IcMenuItem label="Delete" variant="destructive" />
+        </IcPopoverMenu>
+      </>
+    </div>
+  );
+};
+
+export const PopoverMenuWithVariants = () => {
+  const buttonEl = useRef<any>();
+  const popoverEl = useRef<any>();
+  const handleClick = () => {
+    popoverEl.current.open = buttonEl.current.dropdownExpanded;
+  };
+  const handleClose = () => {
+    buttonEl.current.dropdownExpanded = popoverEl.current.open;
+  };
+
+  return (
+    <>
+      <IcButton
+        id="popover-button"
+        dropdown
+        variant="primary"
+        onClick={() => handleClick()}
+        ref={buttonEl}
+      >
+        Button
+        <SlottedSVG
+          slot="icon"
+          xmlns="http://www.w3.org/2000/svg"
+          width="16"
+          height="16"
+          fill="currentColor"
+          class="bi bi-three-dots-vertical"
+          viewBox="0 0 16 16"
+        >
+          <path d="M9.5 13a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0zm0-5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0zm0-5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0z" />
+        </SlottedSVG>
+      </IcButton>
+
+      <IcPopoverMenu
+        anchor="popover-button"
+        aria-label="popover"
+        onIcPopoverClosed={() => handleClose()}
+        ref={popoverEl}
+      >
+        <IcMenuItem label="Copy" disabled />
+        <IcMenuItem label="Paste" keyboardShortcut="Cmd + V" />
+        <IcMenuItem
+          label="Edit"
+          description="This is used to edit the original."
+        />
+        <IcMenuItem label="Show/Hide" variant="toggle" />
+        <IcMenuItem label="Find icons">
+          <SlottedSVG
+            slot="icon"
+            xmlns="http://www.w3.org/2000/svg"
+            height="24px"
+            viewBox="0 0 24 24"
+            width="24px"
+          >
+            <path d="M0 0h24v24H0V0z" fill="none" />
+            <path d="M17.65 6.35C16.2 4.9 14.21 4 12 4c-4.42 0-7.99 3.58-7.99 8s3.57 8 7.99 8c3.73 0 6.84-2.55 7.73-6h-2.08c-.82 2.33-3.04 4-5.65 4-3.31 0-6-2.69-6-6s2.69-6 6-6c1.66 0 3.14.69 4.22 1.78L13 11h7V4l-2.35 2.35z" />
+          </SlottedSVG>
+        </IcMenuItem>
+        <IcMenuItem label="Delete" disabled variant="destructive" />
+        <IcMenuItem label="Log out" variant="destructive" />
+      </IcPopoverMenu>
+    </>
+  );
+};
+
+export const PopoverWithMenuGroups = () => {
```

Comment:
Might want to swap this example to have a dropdown button instead of icon button, and have the open and close functionality the same as the other popover examples for consistency